### DISCUSSION TITLE
local MSDF font

### DISCUSSION THREAD

**Discussion by lihong-xmuseum** [mozilla/hubs#4595] [Hubs Cloud Questions] [answered]
Hello, I want to know how to reference the ![微信图片_20210901180447](https://user-images.githubusercontent.com/69456381/131653052-e4a945ea-158d-49fc-b728-2a369be52b78.png) here?

**Comment by pattersonbl2** [comment, marked as answer]
Would this be a custom client ?  if so we may not have the right answers for you but our community discord may be able to help.
